### DISCUSSION TITLE
HTMLRewriter and Next.js 13

### DISCUSSION THREAD

**Discussion by dberhane** [cloudflare/next-on-pages#296] [General]
I need to use HTMLRewriter in my Next.js 13 on Cloudflare Page. How do I enable it in my local development and when I deploy to the Cloudflare Pages platform?

**Comment by dberhane** [comment]
HTMLRewriter support is built-in to the Next.js 13 on Cloudflare Pages. I just needed to get my local development to use the build command, "npx @cloudflare/next-on-pages@1" and HTMLRewritter in local development work out of the box.

**Comment by dario-piotrowicz** [comment]
Awesome, I'm glad you solved it :smiley:
